src/Sarif/Core/SarifLog.cs

```diff
@@ -87,7 +87,7 @@ public static SarifLog Load(Stream source, bool deferred = false)
 
             if (!fileSystem.FileExists(filePath))
             {
-                throw new ArgumentException(nameof(filePath));
+                throw new ArgumentException($"File path does not exist: '{filePath}'", nameof(filePath));
```

Comment:
>ArgumentException [](http://example.com/codeflow?start=26&length=17)

This is a very comment API use issue, this exception type does not accept a method overload that only references the parameter name (unlike, say, NullReferenceException).